src/AnomalyDetectors/OneClassSVM.php

```diff
@@ -185,8 +185,10 @@ public function train(Dataset $dataset) : void
 
         $data = [];
 
-        foreach ($dataset->samples() as $i => $sample) {
-            $data[] = array_merge([1], $sample);
+        foreach ($dataset->samples() as $sample) {
```

Comment:
So did array_unshift() turn out to be faster?

Is it necessary to assign the sample to an intermediate variable or would

```php
foreach ($dataset->samples() as $sample) {
    array_unshift($sample, 1);

    $data[] = $sample;
}
```

work here as well?

Reply:
It is faster yes. And the rest works also, please see the next commit